commons-numbers-examples/examples-jmh/src/main/java/org/apache/commons/numbers/examples/jmh/gamma/ErfPerformance.java

```diff
@@ -146,6 +146,16 @@ protected double[] createNumbers(SplittableRandom rng) {
      * Contains an array of numbers and the method to compute the error function.
      */
     public abstract static class FunctionData extends NumberData {
+        /** The type of the data.
+         Should be num uniform
+         Declared in ErfPerformance class. **/
+        @Param({NUM_UNIFORM})
+        private String NUM_TYPE;
```

Comment:
The previous benchmark had a single variable `type` in each `@State` class. By splitting this into two you will cause JMH to run the benchmark with more variables. JMH creates benchmarks as the cross-product of all variables. Thus one of the states will not have `1 * 2` benchmarks where previously it would be 1. This is a waste of resources. It will also create two columns in the results where the two columns have redundant information as child classes use only one of the variables.